hackclub.com.yaml

```diff
@@ -426,6 +426,10 @@ archived.guide:
   ttl: 600
   type: CNAME
   value: hackclub-guide.netlify.com.
+arson:
```

Comment:
Please remove from hackclub.com

Reply:
> Please remove from hackclub.com

i have removed it from hackclub.com.yaml and added the value it to dino.icu.yaml in it's correct alphabetical spot